src/BaGetter.Azure/Storage/BlobStorageService.cs

```diff
@@ -79,7 +79,7 @@ public async Task DeleteAsync(string path, CancellationToken cancellationToken)
         {
             await _container
                 .GetBlockBlobReference(path)
-                .DeleteIfExistsAsync(cancellationToken);
+                .DeleteIfExistsAsync();
```

Comment:
This starts to look planned, anything about cancellation tokens I should know?

Reply:
The api changed: They discarded the CT as single argument /facepalm